api/views.py

```diff
@@ -153,11 +153,10 @@ def import_declaration_del(request, import_declaration_id):
 # Create multiple areas at once from CSV-like text
 def add_multi_areas(request, parent_id, structure_id):
     if request.method == 'POST' and request.user.is_authenticated:
-        area_data = request.POST.get('area_csv_data')
-        areastring = io.StringIO(area_data)
-        reader = csv.reader(areastring)
         parent = Area.objects.get(id=parent_id)
-        for row in reader:
+        lines = request.POST.get('area_csv_data').split('\n')
```

Comment:
Approving for now, but think it would be better to keep using `csv.reader` with a specified delimiter, and use quoting in `tableToCSV` to handle delimiters within URL. Seems quite possilbe that URLs might also contain a `|` character.

Reply:
From the brief googling Ive done it seems like `|` isn't used in urls. I dropped the `csv.reader` because I couldn't figure out how to specify a delimiter or escape values.